.github/workflows/lint.yml

```
@@ -0,0 +1,31 @@
+      contents: read
+      pull-requests: read
+      packages: read
+      statuses: write
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v4
+        with:
+          # Super-linter needs the full git history to get the list of files that changed across commits
+          fetch-depth: 0
+      - name: Super-linter check
+        uses: super-linter/super-linter@v7.0.0 # x-release-please-version
```

yamllint

[warning] too few spaces before comment

(comments)